models/src/main/scala/coop/rchain/models/ProtoM.scala

```diff
@@ -228,7 +230,9 @@ object ProtoM extends DescriptorPimps {
         case FIXED32  => computeFixed32Size    (field.getNumber, value.asInstanceOf[Int])
         case BOOL     => computeBoolSize       (field.getNumber, value.asInstanceOf[Boolean])
         case STRING   => computeStringSize     (field.getNumber, value.asInstanceOf[String])
-        case GROUP    => computeGroupSize      (field.getNumber, value.asInstanceOf[MessageLite])
+        case GROUP    => throw new UnsupportedOperationException(
+          s"Groups are not deprecated, got $value of type ${value.getClass}"
```

Comment:
I now think that both `case`-es for `GROUP` (here and above) should `throw new UnsupportedOperationException("Groups are not supported, becuase they're deprecated in protobuf")